### DISCUSSION TITLE
Change message of the response body in the endpoint

### DISCUSSION THREAD

**Discussion by gikpro** [fastapi-users/fastapi-users#1095] [Q&A]
Hello ! what is the best way to add new message (more details) to the response body ? Example: In the login endpoint if a user is not authorized i need to add the reason.
```
{msg: Not authorized,
detail: Your account is not actived yet}
```  

Thanks ! :)

**Comment by frankie567** [comment]
For now, the best way to do this is to re-implement the login route yourself: https://github.com/fastapi-users/fastapi-users/blob/master/fastapi_users/router/auth.py

**Reply by gikpro** [reply]
Thanks for your reply. I will try to do that. ;)
